JSON_for_IO-Link.yaml

```diff
@@ -1743,7 +1750,9 @@ paths:
       tags:
         - masters
       summary: Write application specific identification to a Master.
-      description: Write application specific identification to a Master.
+      description: >
+        Write application specific identification to a Master.
+        Only required, if at least one of the tags is implemented.
```

Comment:
Master related tags has been introduced by this specification. I think it does make sense to change them to be mandatory.